### DISCUSSION TITLE
Storage download/upload with a progressbar

### DISCUSSION THREAD

**Discussion by slavicdev34** [supabase/supabase#1032] [Questions]
Hi guys,

is it possible to show a progress bar during the uploading and downloading files from the supabase storage?

Thanks.

**Comment by fenos** [comment]
It is not currently possible, but it is on our radar to support it

**Comment by inian** [comment]
You can now listen to progress events with our resumable uploads feature in [Storage v3](https://supabase.com/blog/storage-v3-resumable-uploads). Here are the docs on [resumable uploads](https://supabase.com/docs/guides/storage/uploads#resumable-upload).
